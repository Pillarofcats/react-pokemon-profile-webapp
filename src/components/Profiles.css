.Profiles-container {
  min-height: 80vh;
  padding-top: 180px;
  display: flex;
  align-items: center;
  justify-content: center;
  flex-wrap: wrap;
}