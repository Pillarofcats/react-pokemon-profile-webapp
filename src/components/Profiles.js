import React from 'react';
import Profile from './Profile.js';
//styles
import "./Profiles.css"
//Images
import pokemonImages from "../pokemon-images/pokemon-images.js";

const Profiles = ({pokemon}) => {

  return (
    <div className = 'Profiles-container'>
      {
        pokemon.map((aPokemon, index) => {
          return (
            <Profile
              key = {index}
              id = {index}
              image = {pokemonImages[aPokemon.name]}
              name = {aPokemon.name}
            />
          )
        })
      }
    </div>
  )
};

export default Profiles;