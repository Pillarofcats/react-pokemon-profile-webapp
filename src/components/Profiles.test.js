import React from "react";

//React Enzyme Testing 
import { shallow, mount, render } from 'enzyme';
import Profiles from './Profiles.js';

//Snapshot with Jest
it("Snapshot with Jest", () => {
  const mockPokemonList = [
    {
      name: "bulbasaur"
    }
  ];

  expect(shallow(<Profiles pokemon = {mockPokemonList} />)).toMatchSnapshot();
});