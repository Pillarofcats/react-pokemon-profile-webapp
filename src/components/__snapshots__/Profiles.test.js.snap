// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot with Jest 1`] = `ShallowWrapper {}`;
